reference/commands/custom_commands.rst

```diff
@@ -0,0 +1,139 @@
+.. _examples_extensions_custom_commands:
+
+Custom commands
+=================
+
+Since Conan 2.0, it's so easy to create your own Conan commands.
+
+Location and naming
+--------------------
+
+All the custom commands must be located in ``[YOUR_CONAN_HOME]/extensions/commands/`` folder. If _commands_ one is not created yet,
```

Comment:
how do I locate `YOUR_CONAN_HOME`? some link to another page would be helpful here.